### ISSUE TITLE
5 and 10mhz bandwith support. ls it possible?

### ISSUE THREAD

**Issue by cyclonecyc** [Byzantium/Byzantium#231]
Being able to change channel bandwidth would help the amature community be able extend the range of a more defined network.

**Comment by haxwithaxe** [comment]
it is possible. iw is able to do this, but it requires the device to support it.
to check compatibility:

```
# iw phy list
```

under the band on the device you want to use, under "Supported commands:" check for "set_channel".

```
iw phy1 set channel 1
```

where 1=HT20, 2=HT40+, 3=HT40- (i think those are the right numbers but not entirely sure)

reference: http://wireless.kernel.org/en/users/Documentation/iw#Setting_channel_with_iw

you can also change the bit rate (can help with range) with iw

**Comment by cyclonecyc** [comment]
Thanks I will give it a try but I dont see where the channel bandwidth can change to 5mhz instead of 20mhz.
The purchased software from Staros that I am using does this. dd-wrt and openwrt seem to support the 5 and 10mhz bandwidth. The linux kernel I think has been updated in 3.11.

**Comment by haxwithaxe** [comment]
yeah. the drivers/firmware need to support it and the only common change people would make is to go wider so i haven't seen anything allowing narrower in laptops but i'm sure there is stuff out there that supports it.
let us know if you find anything specific that works. others will be interested.
